PublishParasoftTestResults/xsl/sarif.xsl

```diff
@@ -404,12 +405,79 @@
                 </xsl:if>
             </xsl:when>
             <xsl:otherwise>
-                <xsl:value-of select="@locFile" /><xsl:text>"</xsl:text>
+                <xsl:choose>
+                    <xsl:when test="$isCPPProReport">
+                        <!-- For cppTest professional report, "prjModule" attribute is not present.
+                            Project name presents twice in the full source path and needs to be removed.-->
+                        <xsl:value-of select="$locationMap(@locFile)" /><xsl:text>"</xsl:text>
+                    </xsl:when>
+                    <xsl:when test="/ResultsSession/@toolId = 'dottest'">
+                        <!-- For DotTest report, project name prefix is missing in "resProjPath" of "Loc".
+                               As a result, the root folder will be removed from full source path. -->
```

Comment:
Please update the second sentence because it's not applied anymore. We didn't remove root folder here but use @locFile instead.